terraform/projects/app-ecs-services/prometheus-service.tf

```diff
@@ -7,6 +7,21 @@
 *
 */
 
+variable "prom_cpu" {
```

Comment:
Will need to run the `make docs` command to generate new docs as we have new variables please

Reply:
ok